test/e2e/#smoke.test.js

```diff
@@ -13,24 +13,28 @@ const users = [
     name: 'Police user',
     role: policeUser,
     username: 'End-to-end Police',
```

Comment:
Do we still need both `username` and `displayName`? I feel like it was only used for the smoke tests that you've changed to use `displayName`?